components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -389,8 +389,7 @@ class NewTabPage extends React.Component<Props, State> {
   }
 
   startRewards = () => {
-    chrome.braveRewards.saveAdsSetting('adsEnabled', 'true')
-    chrome.braveRewards.setAutoContributeEnabled(true)
+    chrome.braveRewards.saveOnboardingResult('opted-in')
```

Comment:
@zenparsing I'm assuming we want to do a real opt-in here, but let me know if that's wrong.